app/donate/page.tsx

```diff
@@ -0,0 +1,430 @@
+      foodCategory: "",
+      specialHandling: "",
+      quantity: 0,
+      foodImages: []
+    },
+  });
+
+  function onSubmit(values: any) {
+    console.log(values);
+
+    // make api call to save donation details in mongodb 
+    if (foodType === "Cooked Food") { 
+      cookedForm.reset({
+        foodName: "",
+        timePrepared: "",
+        specialHandling: "", 
+        numberOfServings: 0
+      })
+    } else { 
+      nonCookedForm.reset({
+        foodName: "",
+        expiryDate: "",
+        foodCategory: "",
+        specialHandling: "",
+        quantity: 0
+      });
+      setSelectedCategory(""); 
+    }
+
+  }
+  const [selectedCategory, setSelectedCategory] = useState("");
+  const [foodType, setFoodType] = useState("Non-Cooked Food");
+  const [previewImages, setPreviewImages] = useState<string[]>([]);
+
+
+  const handleFileChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    const files = Array.from(e.target.files || []);
+    if (files) {
+      const previews = files.map((file) => URL.createObjectURL(file));
+      setPreviewImages(previews);
+      // cookedForm.setValue("foodImages", files); // Store files in the form state
+    }
+  };
+
+
+  return (
+    <div className="bg-gray-100 min-h-screen p-8">
+      {/* Navigation Bar */}
+      <header className="flex justify-between items-center mb-8">
+        <div className="text-2xl font-bold">FoodToGive</div>
+        <nav className="space-x-4">
+          <a href="/" className="text-gray-700 hover:text-gray-900">Home</a>
+          <a href="/about" className="text-gray-700 hover:text-gray-900">About Us</a>
+          <a href="/donate" className="text-gray-700 hover:text-gray-900">Donate</a>
+          <a href="/account" className="text-gray-700 hover:text-gray-900">Account</a>
+        </nav>
+      </header>
+
+
+      {/* Form Content */}
+      <section className="bg-white rounded-lg shadow-lg p-12 mb-12 flex justify-center">
+        <div className="flex flex-col items-center w-full max-w-4xl">
+        <h1 className="text-2xl font-bold mb-2">Donate Food</h1>
+        <p className="text-sm text-gray-700 mb-7">Help us make a difference, one meal at a time. Share your food today!</p>
+
+        <div className="flex justify-center space-x-4 mb-6">
+
+          <button
+            className={`px-4 py-2 rounded-md font-semibold ${
+              foodType === "Non-Cooked Food" 
+                ? "bg-green-600 text-white"
+                : "bg-gray-100 text-gray-700"
+            }`}
+            onClick={() => setFoodType("Non-Cooked Food")}
+          >
+            Non-Cooked Food
+          </button>
+          <button
+            className={`px-4 py-2 rounded-md font-semibold ${
+              foodType === "Cooked Food" 
+                ? "bg-green-600 text-white"
+                : "bg-gray-100 text-gray-700"
+            }`}
+            onClick={() => setFoodType("Cooked Food")}
+          >
+            Cooked Food
+          </button>
```

Comment:
Should we look into standardising the overall styles for buttons?

The Button component provided by shadcn can be modified to fit in with the styles we want our buttons to look like.

Reply:
Ooh yes we should standardise that...I'll change to use the Button component by shadcn! 
maybe we can use black for any submit buttons (which i think we're all using right now) and possibly green for those that involve toggling (like my cooked vs non-cooked one, and your admin/beneficiary/donor one) ? what do u think

Reply:
Yup sounds good! I will see what I can do to change the colour of the tabs to green!